waymo_raw_data_to_argoverse.py

```diff
@@ -30,205 +40,398 @@
-				if save_poses:
-					dump_pose(city_SE3_egovehicle, timestamp_ns, log_id)
-
-				if save_images:
-					camera_name = CAMERA_NAMES[tf_cam_image.name]
-					img = tf.image.decode_jpeg(tf_cam_image.image)
-					img_save_fpath = f'TEST_RAW_DATA/logs/{log_id}/{camera_name}/{camera_name}_{timestamp_ns}.jpg'
-					assert not Path(img_save_fpath).exists()
-					check_mkdir(str(Path(img_save_fpath).parent))
-					imageio.imwrite(img_save_fpath, img)
-					img_count += 1
-					if img_count % 100 == 0:
-						print(f"\tSaved {img_count}'th image for log = {log_id}")
-
-				
-				# pose_save_fpath = f'logs/{log_id}/poses/city_SE3_egovehicle_{timestamp_ns}.json'
-				# assert not Path(pose_save_fpath).exists()
-				# save_json_dict(pose_save_fpath)
+def check_mkdir(dirpath: str) -> None:
+    """ """
+    if not Path(dirpath).exists():
+        os.makedirs(dirpath, exist_ok=True)
 
 
-def form_calibration_json(calib_data):
-	"""
-	Argoverse expects to receive "egovehicle_T_camera", i.e. from camera -> egovehicle, with
-		rotation parameterized as quaternion.
-	Waymo provides the same SE(3) transformation, but with rotation parmaeterized as 3x3 matrix
-	"""
-	calib_dict = {
-		'camera_data_': []
-	}
-	for camera_calib in calib_data:
-
-		cam_name = CAMERA_NAMES[camera_calib.name]
-		# They provide "Camera frame to vehicle frame."
-		# https://github.com/waymo-research/waymo-open-dataset/blob/master/waymo_open_dataset/dataset.proto
-		egovehicle_SE3_camera = np.array(camera_calib.extrinsic.transform).reshape(4,4)
-		x, y, z = egovehicle_SE3_camera[:3,3]
-		egovehicle_R_camera = egovehicle_SE3_camera[:3,:3]
-
-		assert np.allclose( egovehicle_SE3_camera[3], np.array([0,0,0,1]) )
-		egovehicle_q_camera = rotmat2quat(egovehicle_R_camera)
-		qw, qx, qy, qz = egovehicle_q_camera
-		f_u, f_v, c_u, c_v, k1, k2, p1, p2, k3 = camera_calib.intrinsic
-
-		cam_dict = {
-			'key': 'image_raw_' + cam_name,
-			'value': {
-				'focal_length_x_px_': f_u,
-				'focal_length_y_px_': f_v,
-				'focal_center_x_px_': c_u,
-				'focal_center_y_px_': c_v,
-				'skew_': 0,
-				'distortion_coefficients_': [0,0,0],
-				'vehicle_SE3_camera_': {
-					'rotation': {'coefficients': [qw, qx, qy, qz] },
-					'translation': [x,y,z]
-				}
-			}
-		}
-		calib_dict['camera_data_'] += [cam_dict]
-
-	return calib_dict
-
-
-def dump_pose(city_SE3_egovehicle, timestamp, log_id):
-	""" """
-	x,y,z = city_SE3_egovehicle[:3,3]
-	R = city_SE3_egovehicle[:3,:3]
-	assert np.allclose( city_SE3_egovehicle[3], np.array([0,0,0,1]) )
-	q = rotmat2quat(R)
-	w, x, y, z = q
-	pose_dict = {
-		'rotation': [w, x, y, z],
-		'translation': [x,y,z]
-	}
-	json_fpath = f'TEST_RAW_DATA/pose_logs/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
-	check_mkdir(str(Path(json_fpath).parent))
-	save_json_dict(json_fpath, pose_dict)
-
-
-
-if __name__ == '__main__':
-
-	save_images = True
-	save_poses = True
-	save_calibration = True
-
-	main(save_images, save_poses, save_calibration)
+def save_json_dict(
```

Comment:
we can remove this function and just import 
```python
from argoverse.utils.json_utils import save_json_dict
```

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/5aa62285b6622c6372a097bc05a24214f4f59757)